bmt/bmt_test.go

```diff
@@ -346,11 +362,16 @@ func testHasherCorrectness(bmt *Hasher, hasher BaseHasherFunc, d []byte, n, coun
 	if len(d) < n {
 		n = len(d)
 	}
-	binary.BigEndian.PutUint64(span, uint64(n))
+	binary.LittleEndian.PutUint64(span, uint64(n))
```

Comment:
why LittleEndian suddenly?

Reply:
I can't remember off the top of my head, but at least it's same as in `storage/types.go`?